composer.json

```diff
@@ -31,6 +31,7 @@
 		"jakub-onderka/php-parallel-lint": "^1.0",
 		"localheinz/composer-normalize": "^1.1.0",
 		"phing/phing": "^2.16.0",
+		"php-decimal/stubs": "^1.1",
```

Comment:
What is this for?

Reply:
https://travis-ci.org/phpstan/phpstan/jobs/534308312#L491

We can write a different extension and remove this one instead.

Reply:
Do that for some `TestDecimal` class that you create under `tests/`...